integrations/operator/Dockerfile

```diff
@@ -1,6 +1,37 @@
-ARG BUILDBOX
 # BUILDPLATFORM is provided by Docker/buildx
-FROM --platform=$BUILDPLATFORM $BUILDBOX as builder
+FROM --platform=$BUILDPLATFORM docker.io/debian:11 as builder
+ARG BUILDARCH
+
+## Install dependencies.
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    curl \
+    ca-certificates \
+    unzip \
+    build-essential \
+    gcc-arm-linux-gnueabi \
+    binutils-arm-linux-gnueabi \
+    gcc-aarch64-linux-gnu \
+    binutils-aarch64-linux-gnu
+
+# Install Go.
+ARG GOLANG_VERSION
```

Comment:
It looks like this has broken the drone tag build pipeline. Drone runs `docker buildx create` with this `Dockerfile`, so it is not enough to just update the `Makefile` with the build args. We also need to ensure that dronegen generates an appropriate set of build args too.

https://github.com/gravitational/teleport/blob/2f50a7f48062b21666e498243372d461eb02d61e/dronegen/container_image_products.go#L97

https://github.com/gravitational/teleport/blob/2f50a7f48062b21666e498243372d461eb02d61e/.drone.yml#L8087